package.json

```diff
@@ -71,8 +73,10 @@
     "jscs": "^2.8.0",
     "jshint": "^2.8.0",
     "mocha": "^2.1.0",
+    "phantomjs": "^1.9.19",
```

Comment:
> PS. Right now my PR is broken because of the change in jscs they released something new (in their dep-tree) and existing version 2.8.0 picks it up, unmodified files have issues now.

I can take care of that.

> So if we add extra step to appveyor, rather than requiring humans to do extra step (there is no mentioning of it in the readme) would it work for you? I'll adjust my PR to illustrate the idea.

Won't really work as it would require, manually removing that `devDependency` on appveyor before running. I would prefer to just state that tests need phantomjs installed and add that you could install it using `npm install -g phantomjs` or `brew install phantomjs`.